diff --git a/weboob/browser/filters/standard.py b/weboob/browser/filters/standard.py
index 885798d..15f9484 100644
--- a/weboob/browser/filters/standard.py
+++ b/weboob/browser/filters/standard.py
@@ -231,13 +231,13 @@ class CleanText(Filter):
     Then it replaces all symbols given in the ``symbols`` argument.
 
     >>> CleanText().filter('coucou ')
-    u'coucou'
+    'coucou'
     >>> CleanText().filter(u'coucou\xa0coucou')
-    u'coucou coucou'
+    'coucou coucou'
     >>> CleanText(newlines=True).filter(u'coucou\\r\\n coucou ')
-    u'coucou coucou'
+    'coucou coucou'
     >>> CleanText(newlines=False).filter(u'coucou\\r\\n coucou ')
-    u'coucou\\ncoucou'
+    'coucou\\ncoucou'
     """
 
     def __init__(self, selector=None, symbols='', replace=[], children=True, newlines=True, normalize='NFC', **kwargs):
@@ -457,14 +457,14 @@ class Regexp(Filter):
     >>> from lxml.html import etree
     >>> doc = etree.fromstring('<html><body><p>Date: <span>13/08/1988</span></p></body></html>')
     >>> Regexp(CleanText('//p'), r'Date: (\d+)/(\d+)/(\d+)', '\\3-\\2-\\1')(doc)
-    u'1988-08-13'
+    '1988-08-13'
 
     >>> (Regexp(CleanText('//body'), r'(\d+)', nth=1))(doc)
-    u'08'
+    '08'
     >>> (Regexp(CleanText('//body'), r'(\d+)', nth=-1))(doc)
-    u'1988'
+    '1988'
     >>> (Regexp(CleanText('//body'), r'(\d+)', template='[\\1]', nth='*'))(doc)
-    [u'[13]', u'[08]', u'[1988]']
+    ['[13]', '[08]', '[1988]']
     """
 
     def __init__(self, selector=None, pattern=None, template=None, nth=0, flags=0, default=_NO_DEFAULT):
@@ -698,7 +698,7 @@ class QueryValue(Filter):
     >>> from .html import Link
     >>> f = QueryValue(Link('//a'), 'id')
     >>> f(etree.fromstring('<html><body><a href="http://example.org/view?id=1234"></a></body></html>'))
-    u'1234'
+    '1234'
     """
     def __init__(self, selector, key, default=_NO_DEFAULT):
         super(QueryValue, self).__init__(selector, default=default)
diff --git a/weboob/tools/capabilities/paste.py b/weboob/tools/capabilities/paste.py
index b552b31..576799c 100644
--- a/weboob/tools/capabilities/paste.py
+++ b/weboob/tools/capabilities/paste.py
@@ -89,11 +89,12 @@ def test():
     # all expirations are too high
     assert MockPasteModule({1337: '', 42: '', False: ''}).get_closest_expiration(1) is None
     # we found a suitable lower or equal expiration
-    assert MockPasteModule({1337: '', 42: '', False: ''}).get_closest_expiration(84) is 42
+    assert MockPasteModule({1337: '', 42: '', False: ''}).get_closest_expiration(84) == 42
     assert MockPasteModule({1337: '', 42: '', False: ''}).get_closest_expiration(False) is False
-    assert MockPasteModule({1337: '', 42: ''}).get_closest_expiration(False) is 1337
-    assert MockPasteModule({1337: '', 42: '', False: ''}).get_closest_expiration(1336) is 42
-    assert MockPasteModule({1337: '', 42: '', False: ''}).get_closest_expiration(1337) is 1337
-    assert MockPasteModule({1337: '', 42: '', False: ''}).get_closest_expiration(1338) is 1337
+    print(MockPasteModule({1337: '', 42: ''}).get_closest_expiration(False))
+    assert MockPasteModule({1337: '', 42: ''}).get_closest_expiration(False) == 1337
+    assert MockPasteModule({1337: '', 42: '', False: ''}).get_closest_expiration(1336) == 42
+    assert MockPasteModule({1337: '', 42: '', False: ''}).get_closest_expiration(1337) == 1337
+    assert MockPasteModule({1337: '', 42: '', False: ''}).get_closest_expiration(1338) == 1337
     # this format should work, though of doubtful usage
     assert MockPasteModule([1337, 42, False]).get_closest_expiration(84) is 42
diff --git a/weboob/tools/misc.py b/weboob/tools/misc.py
index 86af905..fc51b37 100644
--- a/weboob/tools/misc.py
+++ b/weboob/tools/misc.py
@@ -79,11 +79,11 @@ def iter_fields(obj):
 def to_unicode(text):
     r"""
     >>> to_unicode('ascii')
-    u'ascii'
+    'ascii'
     >>> to_unicode(u'utf\xe9'.encode('UTF-8'))
-    u'utf\xe9'
+    'utfé'
     >>> to_unicode(u'unicode')
-    u'unicode'
+    'unicode'
     """
     if isinstance(text, unicode):
         return text
